src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -437,7 +437,7 @@
 
 helps['aks remove-connector'] = """
 type: command
-short-summary: Remove the ACI Connector from a managed Kubernetes cluster.
```

Comment:
It looks like this was a bad merge or bad conflict resolution. These "(PREVIEW)" labels should not appear.